src/main/kotlin/us/ihmc/build/IHMCBuildExtension.kt

```diff
@@ -38,9 +35,11 @@ open class IHMCBuildExtension(val project: Project)
    var licenseName: String = "Proprietary"
    var companyName: String = "IHMC"
    var maintainer: String = "Rosie (dragon_ryderz@ihmc.us)"
-   
-   private lateinit var nexusUsername: String
-   private lateinit var nexusPassword: String
+
+   private var ihmcNexusUrl = "https://nexus.ihmc.us"
+   private var ihmcNexusUsername: String
+   private var ihmcNexusPassword: String
```

Comment:
Why the rename with the ihmc prefix?

Reply:
Because it's specifically ihmc's Nexus instance